XCUITests/FxScreenGraph.swift

```diff
@@ -332,4 +341,10 @@ extension Navigator {
             self.goto(TabTray)
         }
     }
+
+    func changeBrowserView(view: String) {
```

Comment:
@jhugman Do you know why that error happens? would it be correct to use the long call: 

> navigator.changeBrowserView(view: BrowserMenuChoice.desktop.rawValue)